addons/web/static/src/legacy/scss/primary_variables.scss

```diff
@@ -167,3 +167,7 @@ $o-kanban-dashboard-dropdown-complex-gap: 5px !default;
 // Form view
 
 $o-form-view-sheet-max-width: 1140px !default;
+
+// Border-radius
+
+$o-border-radius-lg: .6rem;
```

Comment:
In `bootstrap_overridden.scss` (L67) we set an arbitrary value for `$border-radius`.
I propose to create a `$o-border-radius` variable as well to keep the same structure.